waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/account/AccountsService.kt

```diff
@@ -29,7 +29,10 @@ object AccountsService {
                 WalletServiceManager.createWallet(tenant, registeredUserId)
             }.also { walletId ->
                 //TODO: inject
-                WalletServiceManager.issuerUseCase.add(IssuerDataTransferObject.default(walletId))
+                val issuer = IssuerDataTransferObject.default(walletId)
+                if (issuer != null) {
```

Comment:
add Logging or error handling to avoid silent failures

Reply:
Not finding the issuer is not an error, it is just an optional feature. That's why we don't need Logging or error handling.